src/getdata.js

```diff
@@ -0,0 +1,11 @@
+const key = 'bdc41b55a4c712c6598fce628e1a5254';
+const url = 'https://api.openweathermap.org/data/2.5/weather?q=';
+
+const getweather = async (term) => {
+  const res = await (
+    await fetch(`${url}${term}&units=metric&appid=${key}`)
```

Comment:
- [ ] Please add error handling to your fetch. You can use a [try/catch block](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/try...catch) for that.
Pathwright screenshot:
![error_handling_requirement](https://user-images.githubusercontent.com/16193212/92930978-2af0b800-f419-11ea-923b-888e76eb3e18.png)